debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/MMObjectAccessBarrier_V1.java

```diff
@@ -49,9 +50,18 @@ protected MMObjectAccessBarrier_V1() throws CorruptDataException
 		if(J9BuildFlags.gc_compressedPointers) {
 			try {
 				J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
-				shift = vm.compressedPointersShift().intValue();
+
+				/* use reflection to access compressedPointersShift which will not exist if build is default */
+				Class J9JavaVMPointerClass = Class.forName("com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer"); //$NON-NLS-1$
```

Comment:
`Class` is a rawType - a non-generic form of a generic class.  Its better to use a non-raw type instead, even if it's just `Class<?>`.

The line above `J9JavaVMPointer vm = ...` has an instance of this class.  Can you use `vm.getClass()` instead of Class.forName?

Reply:
`J9JavaVMPointer.class` is a better choice because that's where the method is declared.